### DISCUSSION TITLE
Use environment variable on Create a Release

### DISCUSSION THREAD

**Discussion by caiofaustino** [community/community#25500] [Actions]
<p>I’m trying to create a release step in my Action that gets the TAG name from an Environment variable.</p>
<p>This is my step:</p>
<pre><code class="lang-auto">- name: Create GitHub Release
        uses: actions/create-release@v1
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          commitish: master
          tag_name: v{{ env.PROJECT_VERSION }}
          release_name: Release {{ env.PROJECT_VERSION }}
          body_path: ${{ github.workspace }}/release_notes.md
          draft: false
          prerelease: false
</code></pre>
<p>PROJECT_VERSION was set in a previous step using <code>echo "PROJECT_VERSION=content" &gt;&gt; $GITHUB_ENV</code></p>
<p>The idea is that the tag would end up beint “v” + “PROJECT_VERSION”, so a common <code>v1.0.0</code>.</p>
<p>Oddly enough <code>${{ github.workspace }}</code> is parsed properly but <code> {{ env.PROJECT_VERSION }}</code> isn’t.</p>
<p>Any ideas?</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="caiofaustino" data-post="1" data-topic="135486">
<div class="title">
<div class="quote-controls"></div>
 caiofaustino:</div>
<blockquote>
<pre><code class="lang-yaml">          tag_name: v{{ env.PROJECT_VERSION }}
          release_name: Release {{ env.PROJECT_VERSION }}
</code></pre>
</blockquote>
</aside>
<p>Probably because the <code>$</code> in front of the opening <code>{{</code> is missing in these two lines.</p>

**Comment by caiofaustino** [comment]
<p>Omg I feel so stupid, I should have a better linter or something. Thank!.</p>
